recipes/yacht/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "1.0" %}
+
+package:
+  name: yacht
+  version: {{ version }}
+
+source:
+  url: https://github.com/KoslickiLab/YACHT/releases/download/v{{ version }}/yacht-{{ version }}-py_0.tar.bz2
+  sha256: a7acc8ec92d4fe1f9046070eff430e688e4de6818de23519168d103ae73d4c66
+
+build:
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('yacht', max_pin="x") }}
+
+requirements:
+  host:
+    - python >3.6
+    - pip
+
+test:
+  imports:
+    - yacht
+
+  commands:
+    - yacht --help
+
+about:
+  home: https://github.com/KoslickiLab/YACHT
+  summary: 'YACHT is a mathematically rigorous hypothesis test for the presence or absence of organisms in a metagenomic sample, based on average nucleotide identity (ANI).'
+  license: MIT
+  license_file: LICENSE.txt
```

Comment:
I think you can remove this line. Test failed due to 
```
ValueError: License file given in about/license_file (/opt/recipe/LICENSE.txt) does not exist in source root dir or in recipe root dir (with meta.yaml)
```

Reply:
Deleted